src/hid/joystick_sfml_keyboard.cc

```diff
@@ -0,0 +1,97 @@
+// BoB robotics includes
+#include "hid/joystick_sfml_keyboard.h"
+
+template<class T, class Func>
+bool runIfFound(const std::map<sf::Keyboard::Key, T> m,
```

Comment:
I think this should be a reference :smile:

Reply:
D'oh! It seems I've forgotten about references entirely recently :worried: